### PR TITLE
feature(scale-tests): reorganize scale tests location

### PR DESCRIPTION
**Pull request by soyacz**
Reorganization of scale tests - moving to separate directory.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/features-scale-450-500-cluster-test.jenkinsfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ longevityPipeline(
     region: 'eu-west-1',
     post_behavior_db_nodes: 'destroy',
     test_name: 'grow_cluster_test.GrowClusterTest.test_grow_x_to_y',
-    test_config: '''["test-cases/features/scale-cluster.yaml", "configurations/scale-450-500.yaml"]''',
+    test_config: '''["test-cases/scale/scale-cluster.yaml", "configurations/scale-450-500.yaml"]''',
```
Comment (fruch) [inline jenkins-pipelines/features-scale-450-500-cluster-test.jenkinsfile:12]: Maybe also in that he configuration folder you want a scale folder ?

Comment (soyacz) [reply jenkins-pipelines/features-scale-450-500-cluster-test.jenkinsfile:12]: yes, configurations are quite organized. done. thanks.

**Review by fruch — APPROVED** [review]
LGTM

**Review by fgelcer — DISMISSED** [review]
LGTM..

@roydahan @fruch , don't we want to move the pipeline files into its own folder as well? so we will have the pipelines also organised into subjects, not only the configuration files..

**Comment by fruch** [comment]
> LGTM..
> 
> @roydahan @fruch , don't we want to move the pipeline files into its own folder as well? so we will have the pipelines also organised into subjects, not only the configuration files..

From my POV, I don't really mind one way or the other, but if those are changed
need to make sure
https://github.com/scylladb/scylla-cluster-tests/blob/master/sct.py#L1192

is updated to create the right folder structure.

**Comment by roydahan** [comment]
We don't need to move the pipelines to a different folder, BUT I think we need to rename them and as @fruch mentioned to change the the script that creates the jobs accordingly.

@soyacz let's try to do it ASAP, because we want to start working on 5.0 with Argus and it's better we will have them in Argus with the correct location and category, so we won't have duplications of "old" and "new" jobs.

@bentsi / @k0machi FYI

**Comment by soyacz** [comment]
I've renamed the files and added another group to `create_test_release_jobs` function.
@fruch please check if it looks fine.

We need to remember to fix jenkinsfiles names in master jenkins folder jobs after merging this commit.

**Review by fruch — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]
Looks good.
I'll merge and run the job to recreate 5.0 jobs.

**Comment by roydahan** [comment]
Why checks are failing?

**Pull request merged**